file_browser/src/components/AddDIalog.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from "react";
+import { SystemItem } from "../classes/SystemItem";
+import { FileClass } from "../classes/File";
+import { Folder } from "../classes/Folder";
+import { observer } from "mobx-react-lite";
+import fileSystemStore from "../stores/FileSystemStore";
+
+const options = ["File", "Folder"];
+
+const AddDialog = observer(({
+  onClose
+}: {
+  onClose: (state: boolean) => void;
+}) => {
+
+  enum Selected {
+    FileOption = 0,
+    FolderOption = 1
+  }
```

Comment:
`Options` would be a more appropriate name for the enum, and you can also remove the `Option` suffix from each and every option.